boards/arm/nucleo_wb55rg/doc/nucleowb55rg.rst

```diff
@@ -0,0 +1,267 @@
+.. _nucleo_wb55rg_board:
+
+ST Nucleo WB55RG
+################
+
+Overview
+********
+
+The Nucleo WB55RG board is a multi-protocol wireless and ultra-low-power device
+embedding a powerful and ultra-low-power radio compliant with the Bluetooth®
+Low Energy (BLE) SIG specification v5.0 and with IEEE 802.15.4-2011.
+
+
+- STM32 microcontroller in VFQFPN68 package
+- 2.4 GHz RF transceiver supporting Bluetooth® specification v5.0 and
+  IEEE 802.15.4-2011 PHY and MAC
+- Dedicated Arm® 32-bit Cortex® M0+ CPU for real-time Radio layer
+- Three user LEDs
+- Board connector: USB user with Micro-B
+- Two types of extension resources:
+
+  - Arduino Uno V3 connectivity
+  - ST morpho extension pin headers for full access to all STM32 I/Os
+
+- Integrated PCB antenna or footprint for SMA connector
+- On-board ST-LINK/V2-1 debugger/programmer with SWD connector
+- Flexible power-supply options: ST-LINK USB VBUS or external sources
+- On-board socket for CR2032 battery
+- On-board ST-LINK/V2-1 debugger/programmer with USB re- enumeration capability:
+  mass storage, virtual COM port and debug port
+
+.. image:: img/nucleowb55rg.jpg
```

Comment:
The board image should be in ``boards/arm/nucleo_wb55rg/doc/img/nucleowb55rg.jpg`` and you've got it in ``boards/arm/nucleo_wb55rg/img/nucleowb55rg.jpg``